packages/components/src/List/SectionHeader.css

```diff
@@ -0,0 +1,61 @@
+.sectionHeader {
+  position: sticky;
+  top: 0;
+  padding: var(--space-small) var(--space-base);
+  border-bottom: 1px solid var(--color-blue);
+  background-color: var(--color-white);
+}
+
+.greyBlue,
+.draft,
+.upcoming {
+  border-bottom-color: var(--color-greyBlue--light);
+  color: var(--color-greyBlue--light);
+}
+
+.green,
+.completed,
+.paid,
+.reimbursed,
```

Comment:
I missed these before but, we should probably also pull out these specific classes.

Reply:
@seanhealy What would you consider the specific class types? Remove everything but the `sectionHeader`? Or should we keep the colors and not the statuses?